docs/operations/logging.rst

```diff
@@ -1,7 +1,7 @@
 Configuring logging
 -------------
 
-All logs are sent to ``stdout`` by default. You can change this by `extending the images <development.html/_extending_images>`_. 
+All logs are sent to ``stdout`` by default. You can change this by :ref:`extending the images <extending_images>`.
```

Comment:
When I click this link, takes me to: https://docs.confluent.io/current/
I think something is wrong with the link because it is not taking you to the topic you have linked to here.

Reply:
Where are you clicking the link in? There isn't really a staged HTML preview, you have to build it yourself locally...

Reply:
@JoyceFee1 how are you testing this? Because these docs are in a separate repo (not in the docs repo),  I believe the only way to test these links is to clone the repo https://github.com/confluentinc/cp-docker-images/, and do a local manual `make html` build. When I do that, all these links are working correctly.